src/c++/uds/kernelLinux/tests/RequestQueue_p1.c

```diff
@@ -484,14 +484,14 @@ static void testQuadTime(const char *label,
     td[i].sections  = 4;
     td[i].producer  = producer;
     td[i].qb        = qb;
-    UDS_ASSERT_SUCCESS(uds_create_thread(producerThread, &td[i], "producer",
+    UDS_ASSERT_SUCCESS(vdo_create_thread(producerThread, &td[i], "producer",
```

Comment:
Are we gonna need to change UDS_ASSERT_SUCCESS to VDO_ASSERT_SUCCESS eventually?

Reply:
This fuction is test-only, so there's no need to do that. We could do it if you think it's useful,though.